_data/projects/microkit.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+
+name: microkit
+display_name: Microkit
+description: A simple operating system framework for developing systems on top of seL4
+project_order: 10
```

Comment:
I have no idea what this `project_order` does, changing the value didn't seem to do anything...

Reply:
It should be the position in the nav bar, IIRC. But you might need to update all other values to achieve a specific result. We should do the `10, 20, 30` thing like back in the `Basic` days where programs had line numbers :-)

Reply:
Ok, so I can confirm that it is the position in the lhs nav bar. I can also confirm that this won't be the first entry with a duplicated value for the sort order, so this already a bit broken. I propose to merge this as is and then make another PR to fix the implementation of ordering in that nav bar in general.

Reply:
Okay I'll leave it as is.